File: app/src/main/java/com/example/xianweili/newsapp/NetworkUtil/NetworkUtil.java
package com.example.xianweili.newsapp.NetworkUtil;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

/**
 * Created by xianweili on 7/24/18.
 */

public class NetworkUtil {

    public static Retrofit getRetrofit(){
        return new Retrofit.Builder()
                .baseUrl("https://newsapi.org/v2")
                .addConverterFactory(GsonConverterFactory.create())
                .build();
    }
}

File: app/src/main/java/com/example/xianweili/newsapp/NetworkUtil/NetworkUtil.kt
package com.example.xianweili.newsapp.NetworkUtil

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory
import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory

object NetworkUtil {

    @JvmStatic
    val retrofit: Retrofit
        get() = Retrofit.Builder()
            .baseUrl("https://newsapi.org/")
            .addConverterFactory(GsonConverterFactory.create())
            .addCallAdapterFactory(RxJava2CallAdapterFactory.create())
            .build()
}